ddtrace/tracer.py

```diff
@@ -331,6 +332,10 @@ def configure(
         if (collect_metrics is None and runtime_metrics_was_running) or collect_metrics:
             self._start_runtime_worker()
 
+        if self.log.level <= logging.INFO:
+            f = flare.jsonflare(self)
+            self.log.info("DD_TRACE_STARTUP_LOGS - %s", f)
```

Comment:
Can we make the comment consistent with what suggested in the doc? https://docs.google.com/document/d/146LTvy5WbfcnW5ESb-42sqRF7ACermfMKzvZVdfyJIM/edit?disco=AAAAGrrStA4

`[all the default things] - DATADOG TRACER CONFIGURATION - %s`

Reply:
Ah yes, I seem to have copied the env var lol...